README.md

```diff
@@ -3,3 +3,33 @@
 [![pre-commit hooks](https://github.com/AlexMomot-717/python-project-83/actions/workflows/pre-commit.yml/badge.svg)](https://github.com/AlexMomot-717/python-project-83/actions)
 [![Maintainability](https://api.codeclimate.com/v1/badges/7ad5b8e6d1a22a4fdf34/maintainability)](https://codeclimate.com/github/AlexMomot-717/python-project-83/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/7ad5b8e6d1a22a4fdf34/test_coverage)](https://codeclimate.com/github/AlexMomot-717/python-project-83/test_coverage)
+
+### Setup:
+
+The application will be available at http://0.0.0.0:8000.
+Run the application using following commands:
+
```

Comment:
wdyt, is it necessary to know all the commands you have prepared if they just want to start the application? `make up` should be enough